README.md

```diff
@@ -2,192 +2,128 @@
-
-For a given full-chain intermediate certificate and corresponding key, both as pem files, generate a device certificate and device key valid for 365 days.
-```sh
-omnect-cli identity set-device-certificate -d "device_id" -i <path>/image.wic -c <path>/intermediate_full_chain_cert.pem -k <path>/intermediate_cert_key.pem -D 365
-```
-Note: "device_id" has to match the `registration_id` respectively the `device_id` configured in `config.toml`.
+**Note**: "device_id" has to match the `registration_id` respectively the `device_id` configured in `config.toml`.
 
 See [`config.toml.est.template`](conf/config.toml.est.template) as a corresponding `config.toml` is case of using `EST service`.
 
-### Get full-chain intermediate certificate and key for existing OMNECT PKI
+#### Get full-chain intermediate certificate and key for existing OMNECT PKI
 Please get into contact with us in case you want to use our existing cloud services for device provisioning. We can provide certificate and key file to configure your device.
 
-### Generate your own full-chain intermediate certificate and key
+#### Generate your own full-chain intermediate certificate and key
 In case you intend to use your own certificates (e.g. because you want to use your own `PKI` and/or `EST service`), you can find some information about generating certificate and key here: https://docs.microsoft.com/en-us/azure/iot-edge/how-to-create-test-certificates?view=iotedge-2020-11.
 
-## SSH Tunnel configuration
-
-For the ssh feature, the device requires the public key of the ssh root ca and the principal. The latter should be the device id.
-```sh
-omnect-cli identity set-ssh-tunnel-certificate --image <path>/image.wic --root_ca <path>/ssh_ca.pub --device-principal "device_id"
-```
-
-# Device Update for IoT Hub configuration
-## Inject `du-config.json`
+## Device Update for IoT Hub configuration
+### Inject `du-config.json`
 
 ```sh
 omnect-cli iot-hub-device-update set -c <path>/du-config.json -i <path>/image.wic
 
 Options:
+  -p pack and compress image [xz, bzip2, gzip]
   -b create bmap file
 ```
 
-# Copy Files into or from Image
+## Copy files
+
+Copying files into or from the image is restricted to partitions `boot`, `rootA`, `cert` and `factory`. Destination paths that are not existing will be created on host as well as on image.
 
-Copying files into or from the image is restricted to partions `boot`, `cert` and `factory`.
+### Copy files from image
 
-Note: If you need special permissions on copied files, you have to additionally copy a systemd-tmpfiles.d configuration file which handles these permissions.
-## Read e.g. `boot.scr`
+`omnect-cli` allows copying multiple files from multiple partitions in one command:
 
 ```sh
-omnect-cli file copy-from-image -f /boot.scr -i <path>/image.wic -p boot -d boot.scr
+omnect-cli file copy-from-image -f <partition>:<absolute-path-to-src>,<path-to-dest> -f <partition>:<absolute-path-to-src>,<path-to-dest> -i <path>/image.wic
```

Comment:
for me the format is not really intuitive.
e.g. i would expect something like `-f <absolute-path-to-src>,<partition>:<path-to-dest-in-partion>`